packages/web-react/src/hooks/useScrollControl.ts

```diff
@@ -0,0 +1,40 @@
+import { useEffect } from 'react';
+
+const SCROLLING_DISABLED_CLASSNAME = 'is-scrolling-disabled';
```

Comment:
This is not blocking, but I prefer naming these constants like `CLASSNAME_SCROLLING_DISABLED` and `SELECTOR_DIALOG_OPEN`. We are not consistent with this but it is a most used naming approach in web package JS plugins.